styles/migadu-webmail/catppuccin.user.css

```diff
@@ -0,0 +1,421 @@
+/* ==UserStyle==
+@name Migadu Webmail Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/migadu-webmail
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/migadu-webmail
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/migadu-webmail/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Amigadu-webmail
+@description Soothing pastel theme for Migadu Webmail
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire*", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+@var checkbox hideProfilePictures "Hide Profile Pictures" 0
```

Comment:
This toggle hides profile pictures because they are incredibly annoying to theme but otherwise don't look right so yeah lol.